package.json

```diff
@@ -32,7 +34,6 @@
     "clean:prisma": "rimraf node_modules/.prisma/client && node node_modules/@prisma/client/scripts/postinstall.js",
```

Comment:
need to confirm this is working correctly

Reply:
I see the same warning on `main` branch, so its not related to Yarn version